lib/src/pages/reg_form.dart

```diff
@@ -0,0 +1,218 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:loginform_new/bloc/auth_cubit.dart';
+import 'package:loginform_new/src/pages/dashboard_page.dart';
+
+class FormPage extends StatefulWidget {
+  const FormPage({Key? key}) : super(key: key);
+
+  @override
+  _FormPageState createState() => _FormPageState();
+}
+
+class _FormPageState extends State<FormPage> {
+  TextEditingController emailController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  TextEditingController confirmPasswordController = TextEditingController();
+  GlobalKey<FormState> formState = GlobalKey();
+
+  bool obscureText = true;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Registration Form',
+          textAlign: TextAlign.center,
+        ),
+      ),
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: ListView(
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(top: 15),
+                child: TextFormField(
+                  decoration: InputDecoration(
+                      icon: Icon(Icons.person),
+                      labelText: "Name",
+                      contentPadding: EdgeInsets.all(8),
+                      border: OutlineInputBorder(
+                          borderRadius: BorderRadius.circular(10))),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(top: 15),
+                child: TextField(
+                  keyboardType: TextInputType.number,
+                  decoration: InputDecoration(
+                    icon: Icon(Icons.phone),
+                    labelText: "Enter phone number",
+                    hintText: '+91 0000000000',
```

Comment:
If country code is being added, the the length would be 13 characters. Add country code as a prefix string.